vue/sbc-common-components/src/util/common-util.ts

```diff
@@ -66,6 +66,6 @@ export function appendAccountId (url, accountId = '') {
 
 export function removeAccountIdFromUrl (url, key = ACCOUNT_ID) {
   // replacing account id and formattig ie, removing extra ? or &
-  return url.replace(new RegExp(key + '=\\w+'), '').replace('?&', '?').replace(/\?$/, '')
-    .replace('&&', '&').replace(/&$/, '')
+  return url?.replace(new RegExp(key + '=\\w+'), '')?.replace('?&', '?')?.replace(/\?$/, '')
+    ?.replace('&&', '&')?.replace(/&$/, '')
```

Comment:
Mind. Blown.

lol

Reply:
I think that's right.. probably only need `url?.`